### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by mkralik3**
//skip-ci

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +52,9 @@ Correctly set test.properties and credentials.json (described later)
 
 Added FUSE repositories to maven due to dependencies
 
-Before you import maven project to the IDE, you have to installed *Lombok* and *Cucumber* plugins to the IDE.
+Before you import maven project to the IDE, you have to installed
```
Comment (jstastny-cz) [inline README.md]: installed -> install (since you're editing the line anyway).

File: README.md [hunk 2]
```diff
@@ -50,7 +52,9 @@ Correctly set test.properties and credentials.json (described later)
 
 Added FUSE repositories to maven due to dependencies
 
-Before you import maven project to the IDE, you have to installed *Lombok* and *Cucumber* plugins to the IDE.
+Before you import maven project to the IDE, you have to installed 
+**[Lombok](https://plugins.jetbrains.com/plugin/6317-lombok-plugin)** and
```
Comment (jstastny-cz) [inline README.md]: I'd mark this list that it applies for Idea. If someone uses anything else, they can add a section for IDE of their choice.

File: README.md [hunk 3]
```diff
@@ -388,8 +392,25 @@ If you want to run the tests on the existing remote OpenShift instance, follow t
 	```
 After that, you can run the tests on the remote OpenShift instances. 
 Remember to delete project after testing.
-### Creating a Pull Request
+### Contributing
 
+#### Checkstyle
+Before you start contributing, please set up checkstyle in your IDE.
+For IntelliJ Idea, you have to installed **[CheckStyle-IDEA](https://plugins.jetbrains.com/plugin/1065-checkstyle-idea)* plugin.
```
Comment (jstastny-cz) [inline README.md]: installed->install

File: README.md [hunk 4]
```diff
@@ -388,8 +392,25 @@ If you want to run the tests on the existing remote OpenShift instance, follow t
...
+After that, open Settings > Editor > CodeStyle and click on cogwheel, select Import Scheme > CheckStyle configuration and
+choose checkstyle.xml file from this repository.
+
+**NOTE** IntelliJ Idea doesn't provide auto-format code after saving (because it provides auto-saving). When you want to
```
Comment (jstastny-cz) [inline README.md]: in fact one can configure 'Save Actions' plugin.

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**